package.json

```diff
@@ -2,24 +2,30 @@
   "name": "ml-spectra-processing",
   "version": "6.8.0",
   "description": "Various method to process spectra",
-  "main": "lib/index.js",
-  "module": "src/index.js",
+  "main": "./lib/index.js",
+  "module": "./lib-esm/index.js",
+  "types": "./lib/index.d.ts",
   "files": [
+    "src",
     "lib",
-    "src"
+    "lib-esm"
   ],
   "scripts": {
     "build": "rollup -c && cheminfo-build --root SpectraProcessing",
```

Comment:
This build script allows to generate a browser compatible version.

rollup does not exist in this project anymore. This line should be changed to

`npm run tsc-esm && cheminfo-build --entry lib-esm/index.js --root SpectraProcessing`

Reply:
Resolved